test/boost/schema_change_test.cc

```diff
@@ -670,11 +670,11 @@ future<> test_schema_digest_does_not_change_with_disabled_features(sstring data_
 
 SEASTAR_TEST_CASE(test_schema_digest_does_not_change) {
     std::vector<utils::UUID> expected_digests{
-        utils::UUID("8182496e-4baf-3a07-91e6-caa140388846"),
-        utils::UUID("a65ea746-4d8a-3e5c-8fbf-5f70c14dbcbc"),
-        utils::UUID("a65ea746-4d8a-3e5c-8fbf-5f70c14dbcbc"),
-        utils::UUID("4c138336-4677-3520-8556-4aab007cfedb"),
-        utils::UUID("4c138336-4677-3520-8556-4aab007cfedb"),
+        utils::UUID("2fb5d448-c537-39d1-9384-5166bcdcaa9a"),
```

Comment:
Would you still need this if you have left old tables? I think the usual way of handling this is to introduce a schema feature and adding the new table only once this feature is enabled. @avikivity probably knows better.

Reply:
@haaawk I see you "approved" this patch so I assume you got a satisfactory answer for this? Myself, I don't know the answer, and I really dislike and don't understand this test in general and I think we have been doing it the wrong way, but it's not new to this patch :-( I assume that @kbr- ran this test after his change an it passed.